src/main/resources/default/taglib/t/editForm.html.pasta

```diff
@@ -1,9 +1,10 @@
 <i:arg type="String" name="url" description="Contains the target URL to post to." />
 <i:arg type="String" name="class" default="" description="Contains additional classes to be added to the form tag."/>
+<i:arg type="String" name="id" default="" description="Defines the id."/>
```

Comment:
A more descriptive text might help (i.e. specifies the id to use for the generated form tag) ...